src/pybind/mgr/cephadm/serve.py

```diff
@@ -97,7 +97,15 @@ def serve(self) -> None:
         self.log.debug("serve exit")
 
     def _serve_sleep(self) -> None:
-        sleep_interval = 600
+        sleep_interval = max(
+            30,
+            min(
+                self.mgr.host_check_interval,
+                self.mgr.facts_cache_timeout,
+                self.mgr.daemon_cache_timeout,
+                self.mgr.device_cache_timeout,
+            )
+        )
```

Comment:
this is in the overwhelming majority of clusters going to be 60, due to 

https://github.com/ceph/ceph/blob/40d5c37930f9b7f883c3b6da57be481f1fe6fb6c/src/pybind/mgr/cephadm/module.py#L165-L167

Reply:
Do you mention this because you think it shouldn't loop every 60s?  If not, then we should change the facts setting...

Reply:
I'm not sure where 60s came from, but if we want to loop less often we could increase it.  

I suspect all of these default values will change somewhat once we make a full transition to the exporter agent though...